.github/workflows/bazel-cache-push.yml

```diff
@@ -65,6 +65,14 @@ jobs:
         run: |
           cd lte/gateway
           vagrant ssh -c 'cd ~/magma; bazel build //...' magma
+      - name: Test all with Bazel `bazel test //... --cache_test_results=no`
```

Comment:
why the `--cache_test_results=no`? from my understanding this is not relevant for building caches - and only adds to the runtime.

Reply:
Oh i think you might be right!

> Note that test results are always saved in Bazel's output tree, regardless of whether this option is enabled, so you needn't have used --cache_test_results on the prior run(s) of bazel test in order to get cache hits. The option only affects whether Bazel will use previously saved results, not whether it will save results of the current run.

I will double check locally to see if it affects the cache size, then remove if it doesn't